calls.go

```diff
@@ -167,7 +169,7 @@ func (c *CallService) GetCallsInRange(start time.Time, end time.Time, data url.V
 	d.Del("Page") // just in case
 	if start != Epoch {
 		startFormat := start.UTC().Format(APISearchLayout)
-		d.Set("StartTime>", startFormat)
+		d.Set("StartTime>=", startFormat)
```

Comment:
I just checked once again. There no need to encode it manually.  `http.NewRequest` does the job.